src/commands/command-docs.json

```diff
@@ -32,7 +32,7 @@
                             "type": "string"
                         },
                         "since": {
-                            "description": "the Redis version that added the command (or for module commands, the module version).",
+                            "description": "the server version that added the command (or for module commands, the module version).",
```

Comment:
```suggestion
                            "description": "The server version that added the command (or for module commands, the module version).",
```

Because I'm here I want to fix the mis-capitalization.

Reply:
Yeah, I agree, but it's important to be consistent within a page. I believe it is allowed to start with lowercase if it's not a complete sentence. But it's ugly, so I'd prefer always capitalize after colon. But we should do it in all places then, or leave it for later.

Rendering:

![image](https://github.com/placeholderkv/placeholderkv/assets/273886/51b9aa57-984e-4e9e-b2bc-4c60359e64c1)

Reply:
Let me make a SIM.